src/main/java/com/training/web/basePages/FlipkartBasePage.java

```diff
@@ -0,0 +1,9 @@
+package com.training.web.basePages;
```

Comment:
package name should not be in camel case. It should be all in lower case
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

Reply:
@dheerajsrinath unattended